src/main/java/org/opentripplanner/transit/model/basic/ToStringBuilder.java

```diff
@@ -1,4 +1,4 @@
-package org.opentripplanner.model.base;
+package org.opentripplanner.transit.model.basic;
```

Comment:
I think this is quite useful in other places as well, should it be in a generic util package, rather than in the transit model?